app/templates/records/macros/related_records_block.html

```diff
@@ -0,0 +1,28 @@
+{% macro related_records_block(record) %}
+  <div class="tna-container">
+    <div class="tna-column tna-column--width-1-3 tna-column--full-small tna-column--full-tiny tna-!--margin-top-m">
+      <div class="tna-aside tna-aside--tight tna-background-accent full-height-aside">
+        <strong class="reference related-chip">SERIES</strong>
+        <h3><a href="">Copyright Office: Entry forms, etc</a></h3>
```

Comment:
Could you replace these with the hgroup (headings with supertitles) please?
https://design-system.nationalarchives.gov.uk/styles/typography/#headings-with-supertitles

Reply:
Now changed to hrgroup